### PR TITLE
Redesign 404 page

### PR DESCRIPTION
**Pull request by dalilia**
# Related Issue

- Resolve #404

### CODE REVIEW

**Review by M07am3dM — APPROVED** [review]
Nice touch @dalilia well done, keep up the good work.

**Review by Lilicod — APPROVED** [review]

File: src/pages/404.jsx [hunk 1]
```diff
@@ -1,7 +1,48 @@
...
+    const router = useRouter();
+    const language = router.locale;
+    const { t } = useTranslation("common");
+
+    useEffect(() => {
+        document.body.dir = language == "ar" ? "rtl" : "ltr";
+    }, [language]);
+    return (
+        <Layout>
+            <div className='flex flex-col lg:flex-row items-center justify-center p-4 lg:p-8'>
```
Comment (dzc0d3r) [inline src/pages/404.jsx]: I think adding the following classes will make the page look better `container mx-auto h-screen`

**Review by dzc0d3r — APPROVED** [review]
Amazing job @dalilia 
Thank you

**Review by meryem08 — APPROVED** [review]
Nice Work @dalilia Thank you !!

**Pull request merged**